### PR TITLE
Save app namespacedname in clusterConditions

### PR DESCRIPTION
**Pull request by lennysgarage**
This PR modifies the appsetreport CRD by adding a new `app` field to save the app namespaced name for each cluster. Also working on cleaning up some messages and variable names.

### CODE REVIEW

File: pkg/controller/multiclusterstatusaggregation/multiclusterStatusAggregation_controller.go [hunk 1]
```diff
@@ -343,6 +343,7 @@ func (r *ReconcilePullModelAggregation) generateSummary(appSetClusterStatusMap m
 			Cluster:      cluster.clusterName,
 			SyncStatus:   appSetClusterStatusMap[appset][cluster].SyncStatus,
 			HealthStatus: appSetClusterStatusMap[appset][cluster].HealthStatus,
+			App:          appset.appset.Namespace + "/" + appset.appset.Name,
```
Comment (mikeshng) [inline pkg/controller/multiclusterstatusaggregation/multiclusterStatusAggregation_controller.go:346]:
```suggestion
			App:          appset.appset.Namespace + "/" + appset.appset.Name,
```
The app is in the same namespace as appset but the name of the app is different.
As it gathers the SyncStatus and HealthStatus in the code above, you will need to store the `app`

**Review by mikeshng — CHANGES_REQUESTED** [review]

**Review by mikeshng — APPROVED** [review]
Thank you for doing this @lennysgarage !

/approve

/lgtm

**Comment by xiangjingli** [comment]
@lennysgarage  Just a reminder that another PR is needed for updating the appsetreport CRD in here 

https://github.com/open-cluster-management-io/multicloud-operators-subscription/blob/main/deploy/olm-catalog/multicluster-operators-subscription/manifests/apps.open-cluster-management.io_multiclusterapplicationsetreports.yaml

**Pull request merged**